hooks/useVerifiedLocations.tsx

```diff
@@ -4,166 +4,113 @@ import {
   FOOD_URL,
   TELETEYIT_URL,
   SATELLITE_URL,
+  SAHRA_MUTFAK_URL,
+  PHARMACY_URL,
+  SAFE_LOCATION_URL,
+  DEPREM_IHTIYAC_URL,
 } from "@/utils/constants";
 import useSWR from "swr";
 import { dataFetcher } from "@/services/dataFetcher";
 import dJSON from "dirty-json";
 import { useState } from "react";
 
+type HandleLocationResponseOptions = {
+  extraParamsIcon?: string;
+};
+
+const handleLocationResponse = (
```

Comment:
since all responses are same. tihs function runs when req onSuccess. parses and updates the relevant state.